bin/MessagesBundle_ja.properties

```diff
@@ -218,31 +218,31 @@ lib_in_dir=ディレクトリ内のライブラリ:
 
 Origin = 原点
 Choose_browser = ブラウザ選択
-remove_library_confirm = Really remove library 
-remove_library = Remove library 
-error = Error...
-remove_category_confirm = Really remove category 
-remove_category = Remove category
-remove_macro_confirm = Really remove macro 
-remove_macro = Remove macro
-new_macro_name = Please input new macro name.
-new_category_name = Please input new category name.
-new_library_name = Please input new library name.
-use_sensors_rotate_mirror = Use sensors gestures to rotate and mirror
-No_file_opened = No file opened.
+remove_library_confirm = ライブラリを本当に削除してよろしいですか? 
+remove_library = ライブラリ削除 
+error = エラー...
+remove_category_confirm = カテゴリを本当に削除してよろしいですか? 
+remove_category = カテゴリ削除
+remove_macro_confirm = マクロを本当に削除してよろしいですか? 
+remove_macro = マクロ削除
+new_macro_name = 新しいマクロの名前を入力してください.
+new_category_name = 新しいカテゴリの名前を入力してください.
+new_library_name = 新しいライブラリの名前を入力してください.
+use_sensors_rotate_mirror = 回転と反転にジェスチャーを利用する
+No_file_opened = ファイルを開いていません.
 
-Open_deprecated=Open internal file (deprecated)
-Path_lbl=Path:
+Open_deprecated=内部ファイルを開く (非推奨)
+Path_lbl=パス:
 Default_filename=noname.fcd
 VOID=-
-LABEL_EXPLORER_BUTTON=Select folder
+LABEL_EXPLORER_BUTTON=フォルダを選択
 
-Duplicate=Duplicate
+Duplicate=複製
 
-TopMargin=Top margin (cm)
-BottomMargin=Bottom margin (cm)
-LeftMargin=Left margin (cm)
-RightMargin=Right margin (cm)
-Margins_too_large=Margins are too large!
+TopMargin=上マージン (cm)
+BottomMargin=下マージン (cm)
+LeftMargin=左マージン (cm)
+RightMargin=右マージン (cm)
+Margins_too_large=マージンが大きすぎます!
 Print_outside_regions=Margins are such that the drawing will be printed in regions that are outside the printable area of your printer. Continue anyway?
```

Comment:
Hi @cssho 
Thanks again, I will leave open Issue #27 for that, but the translation is now much more complete, great work! 

Kind regards,
Davide